unity_jobs_simulator/Assets/Scripts/Robot_Controller.cs

```diff
@@ -270,4 +285,27 @@ public void checkAngleErrors()
             }
         }
     }
+
+    public Vector3 getTCPTranspose()
+    {
+        int final = _Arms.Length - 1;
+        ArticulationBody last = _Arms[final];
+        Vector3 worldTranspose = last.transform.position;
+        // Vector3 worldTranspose = Transform.TransformVector(localTranspose);
+        return worldTranspose;
+    }
+
+    public void updateUnityRobotBase_T_UnityRobotTCP()
```

Comment:
Rename this to `CalculateUnityRobotBase_T_UnityRobotTCP` since once the robot moves, the transform is automatically updated outside of this function.